### PR TITLE
Generate Kinfu texture model with attention (BoundingBox) and Ground frame to fix occluded surface

### PR DESCRIPTION
**Pull request by wkentaro**
## Why?

I'd like to create mesh model of objects used in picking task. An application is 6d annotation to create training data.

## How?

Robot looks around objects on table, user annotates bounding box which is static on table in usual case.
In kinfu nodelet, it takes service request with bounding box and ground frame id to generate occluded surface.

### CODE REVIEW

**Comment by k-okada** [comment]
can we merge this?

**Comment by wkentaro** [comment]
I'd like to someone to review this, so please wait for a while.

File: jsk_pcl_ros/src/kinfu_nodelet.cpp [hunk 1]
```diff
@@ -497,7 +538,7 @@ namespace jsk_pcl_ros
 
     // push faces into the texturemesh object
     polygon_1.resize (triangles.polygons.size ());
-    for(size_t i =0; i < triangles.polygons.size (); ++i)
+    for(int i =0; i < triangles.polygons.size (); ++i)
```
Comment (iory) [inline jsk_pcl_ros/src/kinfu_nodelet.cpp]: int

**Comment by wkentaro** [comment]
@iory Thanks! I fixed some of minor issues, so could you please check them?

**Comment by wkentaro** [comment]
## Evaluation

We tested the robustness of this model generation method with arc2017 objects:

**Comment by wkentaro** [comment]
@k-okada Now, we can merge this PR to move forward, so please.

**Comment by k-okada** [comment]
failing need to wait for release?
```
log
[ROSUNIT] Outputting test results to /workspace/ros/ws_jsk_recognition/build/jsk_pcl_ros_utils/test_results/jsk_pcl_ros_utils/rostest-test_add_point_indices.xml
testtest_add_point_indices ... FAILURE!
FAILURE: Package [jsk_tools] for test node [jsk_tools/test_topic_published.py] does not exist
  File "/usr/lib/python2.7/unittest/case.py", line 331, in run
    testMethod()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rostest/runner.py", line 93, in fn
    self.fail(message)
  File "/usr/lib/python2.7/unittest/case.py", line 412, in fail
    raise self.failureException(msg)
--------------------------------------------------------------------------------
[ROSTEST]-----------------------------------------------------------------------
[testtest_add_point_indices][failed]
SUMMARY
 * RESULT: FAIL
 * TESTS: 0
 * ERRORS: 0
 * FAILURES: 1
```

**Comment by wkentaro** [comment]
@k-okada Could you please merge this?

**Pull request merged**